packages/mobile/src/redux/store.ts

```diff
@@ -19,10 +19,10 @@ let lastEventTime = Date.now()
 
 const persistConfig: any = {
   key: 'root',
-  version: 20, // default is -1, increment as we make migrations
+  version: 21, // default is -1, increment as we make migrations
   keyPrefix: `reduxStore-`, // the redux-persist default is `persist:` which doesn't work with some file systems.
   storage: FSStorage(),
-  blacklist: ['geth', 'networkInfo', 'alert', 'fees', 'imports'],
```

Comment:
Why should we persist fees now?

Reply:
I don't know why we weren't doing it before. The stored fee has a `lastUpdated` field that is used to refresh the fee after some time passed, I don't see why restarting the app should change that. I'm ok adding it back though if you think we shouldn't persist it, no strong opinion on my side.

Reply:
Yes no strong opinion, but if we decide to persist it we should be able to recover if it's stored as loading and the app is then killed and restarted.

Reply:
Yeah, I think this is not an issue since we always estimate the fee information if there's stale data in the store, even if it's already set to 'loading' so I don't think the UI can get stuck like that.